src/Search/searchTools.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import uniq from "lodash/uniq";
+import flatten from "lodash/flatten";
+import type { QueryType } from "./Types/QueryType";
+
+// function which normalize input values
+export const transformString = (str: string): string => {
+  return str
+    .toLowerCase()
+    .replace(/ /g, "")
+    .normalize("NFD")
+    .replace(/[\u0300-\u036f]/g, "");
+};
+
+// function which compares data table value with searched value in terms of type and operator
+type FilterByColNameAndOp = {
+  comparedValue: string | number,
+  operator: string,
+  searchedValue: string
+};
+
+export const filterByColNameAndOp = ({
+  comparedValue,
+  operator,
+  searchedValue
+}: FilterByColNameAndOp): boolean => {
+  let res: boolean = false;
+  let formattedOp: string = operator;
+  if (formattedOp === "=") {
+    formattedOp = "==";
+  }
+  switch (typeof comparedValue) {
+    case "boolean":
+      if (formattedOp === "==" || formattedOp === "!=") {
+        return eval(`${comparedValue.toString()} ${formattedOp} ${searchedValue}`);
+      }
+      break;
```

Comment:
Useless break, you have a return

Reply:
If i do not put a "break" eslint will trigger an error

Reply:
use `return false;` instead of break and remove `return false` at the end of file, should do the trick